PWGLF/Tasks/spectraTOF.cxx

```diff
@@ -44,7 +44,9 @@ using namespace o2::framework::expressions;
 struct tofSpectra {
   Configurable<float> cfgNSigmaCut{"cfgNSigmaCut", 3, "Value of the Nsigma cut"};
   Configurable<float> cfgCutVertex{"cfgCutVertex", 10.0f, "Accepted z-vertex range"};
-  Configurable<float> cfgCutEta{"cfgCutEta", 0.8f, "Eta range for tracks"};
+  Configurable<float> cfgCutEtaMax{"cfgCutEtaMax", 0.8f, "Max eta range for tracks"};
+  Configurable<float> cfgCutEtaMin{"cfgCutEtaMin", 0.8f, "Min eta range for tracks"};
```

Comment:
Hi @jloemker can't you remove the absolute value in the check and put `-0.8` in the configurable, it's not obvious that it will be symmetrical

Reply:
Hi @njacazio ,

I included your suggestion and added one more histogram for the eta distribution of the tracks in the collision (processDerived).

Many Thanks !